codemods/react/rename-unsafe-lifecycles/.codemodrc.json

```diff
@@ -5,10 +5,6 @@
   "description": "Renames unsafe React lifecycle methods to their new names with UNSAFE_ prefix",
   "engine": "jscodeshift",
   "private": false,
-  "author": {
-    "name": "Shubhra Agarwal",
-    "email": "agarwalshubhra2002@gmail.com"
-  },
   "arguments": [],
   "include": [
     "**/*.js",
```

Comment:
does this codemod only work for js files? no jsx or ts, tsx files? @shubhraagarwal , @mohab-sameh